Fastpack/Commands.re

```diff
@@ -165,7 +171,7 @@ module Serve = {
                 )
               ) {
               | Error(_) => raise(Context.ExitError(""))
-              | Ok(_ctx) => Lwt.join([devserver, watch()])
+              | Ok(_ctx) => Lwt.join([devserver <?> w, watch()])
```

Comment:
What this operator mean?

Reply:
an alias of `Lwt.choice`, it's a bit confusing as we don't use infix ops much in other parts of code and if we'd use then the whole expression would look like `devserver <?> w <&> watch()`